### PR TITLE
bugfix(new integration): Update Active Campaign Page event mapping for referring domains

### PR DESCRIPTION
**Pull request by yashasvibajpai**
## Description of the change

> Support added for `properties.referring_domain` and` context.page.referring_domain` for Page events 

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by utsabc — DISMISSED** [review]

**Comment by utsabc** [comment]
LGTM

**Review by sanpj2292 — APPROVED** [review]

File: v0/destinations/active_campaign/data/ACPage.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [
   {
     "destKey": "name",
-    "sourceKeys": ["properties.domain", "properties.url", "context.page.url"],
+    "sourceKeys": ["properties.referring_domain", "context.page.referring_domain", "properties.domain", "properties.url", "context.page.url"],
```
Comment (utsabc) [inline v0/destinations/active_campaign/data/ACPage.json:4]: Lets do one more thing lets do URL() parsing for extra set of guard rails

Comment (arnab-p) [reply v0/destinations/active_campaign/data/ACPage.json:4]: Why update the precedence order? We should add the new options at the end

**Review by arnab-p — CHANGES_REQUESTED** [review]

File: v0/destinations/active_campaign/data/ACPage.json (hunk 1 above)
Comment (yashasvibajpai) [reply v0/destinations/active_campaign/data/ACPage.json]: I had initially thought of giving higher precedence to the `referrer_domain` properties. But we have handled the case of a normal url in the metadata so fixing it.

Comment (utsabc) [reply v0/destinations/active_campaign/data/ACPage.json]: I missed this we should maintain this for backward compatibility

Comment (yashasvibajpai) [reply v0/destinations/active_campaign/data/ACPage.json]: I had updated the order.

Comment (utsabc) [reply v0/destinations/active_campaign/data/ACPage.json]: @yashasvibajpai while we are at it can we update the test cases to ga4 standard?

Comment (utsabc) [reply v0/destinations/active_campaign/data/ACPage.json]: New format

**Review by utsabc — APPROVED** [review]

**Review by ItsSudip — APPROVED** [review]

**Pull request merged**